mirrord-operator/README.md

```diff
@@ -20,3 +20,65 @@ If you have a certificate license (usually part of Enterprise offering) you can:
         license.pem: LICENSE_CONTENT
     ```
     then reference it using `license.pemRef` in `values.yaml`
+
+
+### SQS queue splitting
+
+When using EKS we need to allow the operator to modify `sqs` queues
+
+```bash
+export account_id=$(aws sts get-caller-identity --query "Account" --output text)
+
+cat >mirrord-operator-policy.json <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "sqs:*"
+            ],
+            "Resource": [
+                "arn:aws:sqs:*:$account_id:*"
+            ]
+        }
+    ]
+}
+EOF
+
+aws iam create-policy --policy-name mirrord-operator-policy --policy-document file://mirrord-operator-policy.json
+
+export oidc_provider=$(aws eks describe-cluster --name my-cluster --region $AWS_REGION --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
+
+export namespace=mirrord
+export service_account=mirrord-operator
+
+cat >trust-relationship.json <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "arn:aws:iam::$account_id:oidc-provider/$oidc_provider"
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "$oidc_provider:aud": "sts.amazonaws.com",
+          "$oidc_provider:sub": "system:serviceaccount:$namespace:$service_account"
+        }
+      }
+    }
+  ]
+}
+EOF
+
+aws iam create-role --role-name mirrord-operator-role --assume-role-policy-document file://trust-relationship.json --description "Role for SQS splitting for mirrord-operator"
+
+aws iam attach-role-policy --role-name mirrord-operator-role --policy-arn=arn:aws:iam::$account_id:policy/mirrord-operator-role
```

Comment:
When I ran this I got
```
An error occurred (NoSuchEntity) when calling the AttachRolePolicy operation: Policy arn:aws:iam::<redacted_by_t4lz>:policy/mirrord-operator-role does not exist or is not attachable.
```

So for my testing I just attached `mirrorrd-operator-role` to the preexisting `AmazonSQSFullAccess` policy (with ARN `arn:aws:iam::aws:policy/AmazonSQSFullAccess`). I don't know if that's a valid solution for our users as well, or if the limitation to a specific account id [in our custom policy](https://github.com/metalbear-co/charts/pull/46/files#diff-8956cd3c524bf05bc926c83fa9d3ae32b07607e7603a9134a57dcb8fb3f83b0eR42) is important.